PYME/cluster/HTTPDataServer.py

```diff
@@ -679,6 +681,64 @@ def get_h5_part(self, path):
         except IOError:
             self.send_error(404, "File not found - %s, [%s]" % (self.path, path))
             return None
+
+    def get_tabular_part(self, path):
+        """
+
+        Parameters
+        ----------
+        path: str
+            path to an hdf or h5r file on the dataserver computer. Specific parts of the file can be accessed by
+            appending, e.g. /Metadata, for a resulting `path` of something like folder/test.h5r/Metadata. The 'part' of
+            the file returned defaults to FitResults if unspecified, and can additionally include a slice to take before
+            returning. In other words, a `path` of 'folder/test.h5r/FitResultsslice(0,100,None)' will return the first
+            hundred elements of the FitResults table in test.h5r.
+
+        Returns
+        -------
+        f: BytesIO
+            Requested part of the file encoded as bytes
+
+        """
+        from PYME.IO.tabular import NestedRecArraySource
+        from PYME.IO import h5rFile
+        ext = '.h5r' if '.h5r' in path else '.hdf'
+        try:
+            filename, part = path.split(ext)
+            ctype = self.guess_type(filename + ext)
```

Comment:
this should be the content type you are returning, not the content type that the file has - i.e. `application\json` for your implementation